ShortQuestions/hw1.md

```diff
@@ -0,0 +1,72 @@
+# hw1
+
+## 2. practice git using the platform. list the git commands you learned
+
+- git commit
+- git branch, git checkout 
+- git merge
+- git rebase
+
+## 3. What is the basic steps to init a git repo in you local ?
+
+- git bash
+- git init
+
+## 4. How to clone a repo from Github ?
+
+- git clone [link]
+
+## 5. How to create a new branch and checkout to that branch ?
+
+``` 
+git branch newBranch
+git checkout -b newBranch
```

Comment:
git checkout -b <newBranch> is used to:
Create a new branch named <new-branch>, start it at <start-point>, and check the resulting branch out;

You can just use:
git branch <newBranch>
git checkout <newBranch>